### PR TITLE
Added env variables for db connectivity

### PR DESCRIPTION
**Pull request by shadabshaikh0**
Raised PR for this issue #40 
[https://github.com/ACM-VIT/dolos/issues/40](url)

### CODE REVIEW

File: src/database.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 var mysql = require('mysql');
 
 var con = mysql.createConnection({
-  host: "localhost",
-  user: "yourusername",
-  password: "yourpassword"
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD
```
Comment (YashKumarVerma) [inline src/database.js:6]: use or operator to set the default values.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Comment by shadabshaikh0** [comment]
Added default values, please review it.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
Would recommend adding these entries to `.env.example` as well

**Comment by shadabshaikh0** [comment]
Added .env.example

**Review by YashKumarVerma — APPROVED** [review]

**Comment by lopeselio** [comment]
There are some conflicts which need to be resolved @shadabshaikh0

**Pull request merged**